src/Build.UnitTests/Instance/TaskItem_Tests.cs

```diff
@@ -289,7 +289,7 @@ public void Escaping1()
         /// <summary>
         /// Flushing an item through a task run in the task host also should not mess up special characters on the metadata.
         /// </summary>
-#if RUNTIME_TYPE_NETCORE || MONO
+#if RUNTIME_TYPE_NETCORE
         [Fact(Skip = "FEATURE: TASKHOST")]
```

Comment:
Huh, we should be able to turn these on on core now since taskhost works there.

Reply:
I've enabled these tests on Core. Thank you!